### ISSUE TITLE
calculateDirectionsWithCompletionHandler not running

### ISSUE THREAD

**Issue by byroncoetsee** [mapbox/MapboxDirections.swift#13]
Hey there,

Apologies if this "issue" is in the wrong area. If it is, let me know and I will remove it.

When running `directions.calculateDirectionsWithCompletionHandler { (response, error) in ...... }` and adding a breakpoint to the first line inside the completion handler, the code seems to never be executed. It never hits the breakpoint, meaning nothing inside the completion handler gets executed. Can anyone think of possibly why?
I've set the `MBDirectionsRequest` and `MBDirectionsRequest` correctly as far as I can tell.

Here's the full method:
```swift
func calculateTravelInfoMB() {
		let request = MBDirectionsRequest(sourceCoordinate: start_location, destinationCoordinate: stop_location)
		let directions = MBDirections(request: request, accessToken: "pk.xxxxxxxxxxx") //removed for security I suppose
		directions.calculateDirectionsWithCompletionHandler { (response, error) in
			if error != nil {
				NSNotificationCenter.defaultCenter().postNotificationName("tripInfoUnavailable-\(self.name)", object: nil)
				global.showAlert("Hmm...", message: "Something went wrong while trying to get travel information for the route \(self.name). \n\n\(error!.localizedDescription)")
				println("Error getting directions - \(error)")
			} else {
				self.travelInfoMB = response
				NSNotificationCenter.defaultCenter().postNotificationName("tripDataChanged-\(self.name)", object: nil)
			}
		}
	}
```

**Comment by fritz-fritz** [comment]
@byroncoetsee I thought I had the same problem. After digging in further (set a breakpoint in MBDirections) I found that my problem was the accessToken was wrapped Optional and in turn broke the API call and yet no error was being passed back... That might not be the same problem you had but maybe you'll find your mistake by setting the breakpoint in a better place.

**Comment by 1ec5** [comment]
This should be fixed as of #17. The logic had some branches where the completion handler would never get called.
